tests/integration/components/bs-button-group-test.js

```diff
@@ -117,14 +213,43 @@ module('Integration | Component | bs-button-group', function (hooks) {
     // check button's active property
     for (let k = 0; k < 3; k++) {
       assert.equal(
-        this.element.querySelector(`button:nth-child(${k + 1})`).classList.contains('active'),
+        this.element.querySelector(`input[type="radio"]:nth-of-type(${k + 1})`).checked,
```

Comment:
Did you considered not splitting into two separate tests but only adjust this assertion accordingly to Bootstrap version? Something like:

```js
        assert.equal(
          isBootstrap(5)
            ? this.element.querySelector(`input[type="checkbox"]:nth-of-type(${k + 1})`).checked
             : this.element.querySelector(`button:nth-child(${k + 1})`).classList.contains('active'),
          0 === k,
          'only button with same value is active'
        );
```

Similar for the other tests.

I would tend towards using the same test with version dependent code paths unless a feature is only supported for some Bootstrap versions. I fear maintaining the tests gets otherwise more difficult. Especially it would be too easy to miss test coverage for one Bootstrap version.

Reply:
Well, yeah, I was more concerned that tests start to get unreadable, so was more leaning into the other direction. Like when the only difference is a class name, using a helper function as we do seems good, but here we have to write even different code (`. checked` vs. `.classList.contains()`). 

But there is no clear winner here I guess, more preferences...